src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -119,25 +119,53 @@ family:native package:libobjc* category=system
 family:native package:libsystem* category=system
 family:native package:/system/** category=system
 family:native package:/vendor/** category=system
-module:dalvik.system.* category=system
-module:com.android.* category=system
 family:native package:libdispatch.dylib category=system
 family:native package:WebKit category=system
 family:native package:**/libart.so category=system
 package:/apex/com.android.*/lib*/** category=system
 
+
+## java
+# Having a category prevents deriving in_app stack trace rules
+# XXX: Adjust all -app lines in the next PR
```

Comment:
Some of these lines have `-app` instead of a category. I will handle them in the following PR. I prioritized keeping the diff to a minimum, and in some cases, I saw hash changes that I wanted to investigate.